mc2/controllers/docker/forms.py

```diff
@@ -45,6 +50,18 @@ class DockerControllerForm(ControllerForm):
     def clean_port(self):
         return self.cleaned_data['port'] or None
 
+    def clean(self):
+        cleaned_data = super(DockerControllerForm, self).clean()
+        marathon_health_check_cmd = cleaned_data.get(
+            "marathon_health_check_cmd")
+        marathon_health_check_path = cleaned_data.get(
+            "marathon_health_check_path")
+
+        if marathon_health_check_path and marathon_health_check_cmd:
+            raise forms.ValidationError(
+                "You can only specify 1 health check at a time."
```

Comment:
I that something we enforce or the marathon api?

Reply:
It's actually just us.. I'll remove this limitation actually..